src/api/ApiContext.tsx

```diff
@@ -245,16 +245,18 @@ class ApiContextComponent extends React.Component<
 
 		socket.on(
 			StudentSocketEvents.LOGIN,
-			(data: { success: boolean; jobInfo: JobInfo }) => {
+			(data: { success: boolean; jobInfo: JobInfo, err: any }) => {
 				if (!data.success) {
 					const hasEmailInURL = this.props.match.params.email ? true : false;
 
+					let errorMsg;
+					if (data.err.hasOwnProperty("name") && data.err.name === "NotVerified") errorMsg = "Bitte bestätige deine E-Mail zunächst.";
```

Comment:
Where does the error name "NotVerified" come from? Maybe I do something wrong but if I try to get into the queue with an unverified mail address, it still shows "Wir konnten keine Personen mit dieser E-Mail finden."